docs/conventions.rst

```diff
@@ -0,0 +1,95 @@
+
+Desktop files should be named with the application's ID, followed by the ``.desktop`` file extension, and should be placed in ``/app/share/applications/``. For example::
+
+  /app/share/applications/org.gnome.Dictionary.desktop
+
+A minimal desktop file should contain at least the application's *name*, *exec* command, *type* and *icon* name::
+
+  [Desktop Entry]
+  Name=Gnome Dictionary
+  Exec=org.gnome.Dictionary
+  Type=Application
+  Icon=org.gnome.Dictionary
+
+The ``desktop-file-validate`` command can be used to check for errors in desktop files.
+
+AppData files
+-------------
+
+AppData files provide metadata about applications, which is used by application stores (such as Flathub, GNOME Software and KDE Discover). The `Freedesktop AppStream specification <https://www.freedesktop.org/software/appstream/docs/>`_ provides a complete reference for providing AppData.
+
+AppData files should be named with the application ID and the ``.appdata.xml`` file extension, and should be placed in ``/app/share/metainfo/``. For example::
+
+  /app/share/metainfo/org.gnome.Dictionary.appdata.xml
+
+The ``appstream-util validate-relax`` command can be used to check AppData files for errors.
+
+XDG base directories
+--------------------
+
+`XDG base directories <https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html>`_ are a Freedesktop standard which defines standard locations where user-specific application data and configuration should be stored.
+
+By default, Flatpak sets three XDG base directories that should be used by applications for user-specific storage. These are:
+
+===============  =================================  ======================
+Base directory   Usage                              Default location
+===============  =================================  ======================
+XDG_CONFIG_HOME  User-specific configuration files  ~/.var/<app-id>/config
+XDG_DATA_HOME    User-specific data                 ~/.var/<app-id>/data
+XDG_CACHE_HOME   Non-essential user-specific data   ~/.var/<app-id>/cache
+===============  =================================  ======================
+
+For example, GNOME Dictionary should store user-specific data in::
+
+  ~/.var/org.gnome.Dictionary/data
+
+Note that applications can be configured to use non-default base directory locations (see :doc:`sandbox-permissions`).
+
+Filesystem layout
+-----------------
+
+Each Flatpak sandbox, which is the environment in which an application is run, contains the filesystem of the application's runtime. This follows `standard Linux filesystem conventions <https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard>`_. For example, the root of the sandbox contains standard directories, such as:
+
+- ``/bin`` - binaries
+- ``/dev`` - device files
+- ``/lib`` - libraries
+- ``/opt`` - application software packages
```

Comment:
Bad examples. `/opt` doesn't exist at all and `/lib` and `/bin` are just symlinks in the runtime.

Reply:
Maybe just say it is a traditional Linux layout since it doesn't literally conform for FHS. and show `/usr`, `/etc` as examples?